### PR TITLE
Remove build context

### PR DESCRIPTION
**Pull request by adriantpaez**
Remove build context from docker-compose files. The services' source code are stored in the [mock-avs-src](https://github.com/NethermindEth/mock-avs-src) repository, where the user can find the instructions to build the services images.

### CODE REVIEW

File: pkg/health-checker/docker-compose.yml [hunk 1]
```diff
@@ -2,8 +2,7 @@ version: '3.8'
 services:
   main-service:
     container_name: ${MAIN_SERVICE_NAME}
-    build:
-      context: ./src
+    image: mock-avs-health-checker:latest
```
Comment (AntiD2ta) [inline pkg/health-checker/docker-compose.yml]: Use env variables for this as that's the recommended best practice from us

File: pkg/option-returner/docker-compose.yml [hunk 2]
```diff
@@ -2,8 +2,7 @@ version: '3.8'
 services:
   main-service:
     container_name: ${MAIN_SERVICE_NAME}
-    build:
-      context: ./src
+    image: mock-avs-option-returner:latest
```
Comment (AntiD2ta) [inline pkg/option-returner/docker-compose.yml]: Same here

**Review by AntiD2ta — CHANGES_REQUESTED** [review]

File: pkg/health-checker/docker-compose.yml (hunk 1 above)
Comment (adriantpaez) [reply pkg/health-checker/docker-compose.yml]: Done https://github.com/NethermindEth/mock-avs/pull/10/commits/95bade5bcca5c53db1deeb22a7c0dcb24b4effdb

File: pkg/option-returner/docker-compose.yml (hunk 2 above)
Comment (adriantpaez) [reply pkg/option-returner/docker-compose.yml]: Done https://github.com/NethermindEth/mock-avs/pull/10/commits/95bade5bcca5c53db1deeb22a7c0dcb24b4effdb

**Review by AntiD2ta — APPROVED** [review]

**Pull request merged**